Library/Formula/erlang.rb

```diff
@@ -39,11 +39,13 @@ class Erlang < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
-  depends_on "openssl"
   depends_on "unixodbc" if MacOS.version >= :mavericks
   depends_on "fop" => :optional # enables building PDF docs
   depends_on "wxmac" => :recommended # for GUI apps like observer
 
+  depends_on "libressl" => :optional
+  depends_on "openssl" unless build.with? "libressl"
```

Comment:
Not really. If you use `depends_on "openssl" => :recommended`, it makes you to use two flags instead of one to achieve desirable effect without installing OpenSSL:

```
brew install erlang --with-libressl --without-openssl
```

That just makes no sense.